app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TasksController < AuthenticatedController
+  include ProjectScoped
+  include TasksHelper
+
+  skip_before_action :set_project, unless: -> { current_project }
+  skip_before_action :set_nodes, unless: -> { current_project }
+
+  def index
+    @default_columns = ['Title', 'Methodology', 'List', 'Due Date', 'Assigned']
+
+    if current_project
+      @local_storage_key = "project.ce.project_#{current_project.id}.tasks_datatable"
```

Comment:
Another smell, a local storage key is typically a view concern. Why is it in the controller?

```bash
› grep -r local_storage_key app/controllers
```

Didn't yield other instances of this.

Reply:
moved to the view